src/robot_vision.py

```diff
@@ -11,17 +9,14 @@
 import cv2
 from april_tags import euler
 
-#tag size in meter (15 centimeters, eg 6")
-TAG_SIZE = 0.15
+#tag size in meter (16.51 centimeters, eg 6.5")
+TAG_SIZE = 16.51
```

Comment:
In centimeters or meters ?? Seems to be working fine in cm. Will have to change the comment to reflect that